config/crds/infrastructure.cluster.sigs.k8s.io_awsmachines.yaml

```diff
@@ -572,48 +548,34 @@ spec:
         status:
           description: AWSMachineStatus defines the observed state of AWSMachine
           properties:
-            conditions:
-              description: Conditions is a set of conditions associated with the Machine
-                to indicate errors or other status
+            addresses:
+              description: Addresses contains the AWS instance associated addresses.
               items:
-                description: AWSMachineProviderCondition is a condition in a AWSMachineProviderStatus
+                description: NodeAddress contains information for the node's address.
```

Comment:
Do we need to fork/copy the NodeAddress struct?

Reply:
I'm thinking it wouldn't hurt to create our own with inspiration from the NodeAddress struct, since we could very well end up with different requirements down the line.

Reply:
I'll open a PR to CAPI to fork the type, so other providers can use it as well. I can follow-up in a new PR to fixup the CRD/dependency or fix it here, which do you prefer?

Reply:
followup is good